include/simdjson/generic/ondemand/object-inl.h

```diff
@@ -45,10 +45,36 @@ simdjson_really_inline simdjson_result<object> object::start_root(value_iterator
   SIMDJSON_TRY( iter.start_root_object().get(has_value) );
   return object(iter);
 }
-simdjson_really_inline simdjson_result<std::string_view> object::raw_json_token() noexcept {
+simdjson_really_inline error_code object::consume() noexcept {
```

Comment:
Ok. I had omitted the `abandon()` calls. I think that this code should be better.

cc @jkeiser

Reply:
Gah, I can't remember whether abandon() currently means anything. I flipped that one around a lot while I was adding the safety checks.